dali/dfuplus/dfuplus.cpp

```diff
@@ -192,6 +193,15 @@ CDfuPlusHelper::CDfuPlusHelper(IProperties* _globals,   CDfuPlusMessagerIntercep
 
     const char* username = globals->queryProp("username");
     const char* password = globals->queryProp("password");
+    if ( username && *username && (!password || !*password))
+    {
+        VStringBuffer prompt("%s's password: ", username);
+        StringBuffer passw;
+        passw = passwordInput(prompt, passw);
+        globals->setProp("password",passw.str());
+        password = globals->queryProp("password");
```

Comment:
@afishbeck Because I have that password in a local buffer (passw) only and the 'password' is a pointer. When the current block finished then I lost the 'StringBuffer passw' and the 'password' points somewhere undefined place/content.